recipes/tepid/meta.yaml

```diff
@@ -0,0 +1,42 @@
+about:
+  home: 'https://github.com/ListerLab/TEPID'
+  license: 'GPL'
+  summary: 'TEPID uses paired-end illumina sequencing reads to identify novel TE variants.'
+
+build:
+  number: 0
+  skip: True # [not py27 or not linux]
```

Comment:
The build kept failing on osx with error "fatal error: 'stdio_ext.h' file not found" so I skipped it